src/xmllang/xml_parser.h

```diff
@@ -14,11 +14,15 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "xml_parse_tree.h"
 
-int yyxmlparse();
-
 class xml_parsert:public parsert
 {
 public:
+  explicit xml_parsert(message_handlert &message_handler)
+    : parsert(message_handler)
+  {
+    stack.push_back(&parse_tree.element);
```

Comment:
Did this move from somewhere else?

Reply:
No, this is just a part of clear(). Would it be easier to read if I just invoked clear() (at the expense of a tiny amount of redundant work)?

Reply:
Yes. The name `clear` isn't that super then, of course. Maybe `initialize` would have been better.

Reply:
Now using `clear()`.